extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -76,6 +78,7 @@ export async function createAzureFunction(connectionString: string, schema: stri
 			if (!selectedBinding) {
 				return;
 			}
+			TelemetryReporter.createActionEvent(TelemetryViews.CreateAzureFunctionWithSqlBinding, TelemetryActions.startCreateAzureFunctionWithSqlBinding).withAdditionalProperties({ bindingType: selectedBinding.label }).send();
```

Comment:
I feel like this should be at the start of this function instead since that's really where the entire flow is starting. Otherwise we won't be able to tell how many people started this but then exited early due to some issue getting the project file. 

(you could also leave this one here and add another event at the beginning of this function)

If you do that then make sure you're capturing every potential exit location, such as the user opening the docs link or the host file not being found.